src/istgt_lu_disk_vbox.c

```diff
@@ -68,23 +68,28 @@ typedef struct istgt_lu_disk_vbox_t {
 } ISTGT_LU_DISK_VBOX;
 
 #ifndef O_ACCMODE
-#define O_ACCMODE (O_RDONLY|O_WRONLY|O_RDWR)
+#define	O_ACCMODE (O_RDONLY|O_WRONLY|O_RDWR)
 #endif
 
-static void istgt_lu_disk_vbox_error(void *pvUser, int rc, const char *pszFile, unsigned iLine, const char *pszFunction, const char *pszFormat, va_list va) __attribute__((__format__(__printf__, 6, 0)));
+static void istgt_lu_disk_vbox_error(void *pvUser, int rc, const char *pszFile,
+	unsigned iLine, const char *pszFunction,
```

Comment:
continuation lines better to be with 4 spaces, instead of tab. So, it should be
```
static void istgt_lu_disk_vbox_error(void *pvUser, int rc, const char *pszFile,
    unsigned iLine, const char *pszFunction,
```